### PR TITLE
[wip] New test: MIT kerberus 5 (krb5)

### PR DESCRIPTION
**Pull request by srlemke**
- Related ticket: https://progress.opensuse.org/issues/55718
- Needles: None
- Verification run: 
SLE 12.2 http://deathstar.suse.cz/tests/1856
SLE 12.3 http://deathstar.suse.cz/tests/1853
SLE 12.4 http://deathstar.suse.cz/tests/1857
SLE 12.5 http://deathstar.suse.cz/tests/1863
SLE 15.0 http://deathstar.suse.cz/tests/1862
SLE 15.1 http://deathstar.suse.cz/tests/1861

Opensuse TW: http://deathstar.suse.cz/tests/1863
Opensuse Leap 15.1 : http://deathstar.suse.cz/tests/1860

### CODE REVIEW

File: tests/console/krb5.pm [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    assert_script_run './setup-kerberos-server-3.pl --realm openqa.kerberus.org -noy2kc --noinstall';
+
+    #Fix realm address on krb5.cnf file:
+    assert_script_run "sed -i 's/openqa.kerberus.org.kerberus.org/openqa.kerberus.org/g' /etc/krb5.conf";
+
+    #Test Case 1439087: ssh with MIT Kerberos 5:
+    record_info 'Testcase 1439087:';
+
+    #kadmind starts the Kerberos administration server systemd unit:
+    systemctl('restart kadmind', expect_true => 1);
```
Comment (dzedro) [inline tests/console/krb5.pm]: Where did you find `expect_true` ? Can't find it anywhere documented.

File: tests/console/krb5.pm [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    script_run 'rm -f /var/lib/kerberos/krb5kdc/principal*';
+    script_run 'mv /etc/hosts-bkp /etc/hosts';
+    script_run 'mv /etc/hostname-bkp /etc/hostname';
+    script_run 'rm /tmp/krb5';
+    script_run 'hostname `cat /tmp/hostname`';
+    script_run 'rm /tmp/hostname';
+
+    #important log that we have indeed returned former hostname:
+    record_info 'Restored Hostname';
+    assert_script_run 'hostname';
```
Comment (dzedro) [inline tests/console/krb5.pm]: Maybe use validate_script_output for the expected hostname ?

Comment (dzedro) [reply tests/console/krb5.pm]: Or store it at the beginning to variable and compare it here ?

File: tests/console/krb5.pm (hunk 1 above)
Comment (srlemke) [reply tests/console/krb5.pm]: hmm, good question! 
Well there is "expect_false" all around, maybe "expect_true" is just redundant, will check on it.

File: tests/console/krb5.pm (hunk 2 above)
Comment (srlemke) [reply tests/console/krb5.pm]: Sounds a good move!

File: tests/console/krb5.pm (hunk 1 above)
Comment (dzedro) [reply tests/console/krb5.pm]: As far as I know 'expect_true' is the default

**Pull request closed without merging**